File: advanced-programming-in-the-unix-environment-2nd/ch13/ex13.1.c
/*
 * \file ex13.1.c
 * \brief chroot before openlog
 * \author Jianlong Chen <jianlong99@gmail.com>
 * \date 2013-11-19
 */
/* $Id$ */

#include "apue.h"
#include <syslog.h>

int main (int argc, char *argv[])
{
	/*if (chdir ("/usr/local") < 0) {
		err_sys ("chdir failed");
	}*/

	openlog (argv[0], LOG_CONS, LOG_LOCAL0);
	syslog (LOG_ERR, "test log by ex13.1");
	while (1) {
		sleep (3);
	}
	exit (1);
}

File: the-way-to-go/ch13/ex13-1.go
//
// ex13-1.go
// Copyright (C) 2019 Jianlong Chen <jianlong99@gmail.com>
//
// Distributed under terms of the MIT license.
//

package main

import "fmt"

func main() {
	defer func() {
		if e := recover(); e != nil {
			fmt.Printf("Panicing %s\n", e)
		}
	}()

	a := 3 / b()
	fmt.Println(a)
}

func b() int {
	return 0
}
